src/Files.App/Views/WidgetsPage.xaml.cs

```diff
@@ -31,11 +32,7 @@ public sealed partial class WidgetsPage : Page, IDisposable
 		private FileTagsWidget fileTagsWidget;
 		private RecentFilesWidget recentFilesWidget;
 
-		public YourHomeViewModel ViewModel
-		{
-			get => (YourHomeViewModel)DataContext;
-			set => DataContext = value;
-		}
+		public YourHomeViewModel ViewModel { get; set; }
```

Comment:
Can you use dependency injection for this as well?

Reply:
Is it now, this place?

Reply:
YourHomeViewModel's constructor has two params so can't.